solana/ts/scripts/collectedToAnchorToml.ts

```diff
@@ -0,0 +1,14 @@
+import * as fs from "fs";
+
+main();
+
+async function main() {
+    fs.readdirSync("collected").forEach((fn) => {
+        const address = fn.slice(0, -5);
+        console.log(`
+### TODO
```

Comment:
Do we need to leave this TODO in?

Reply:
This TODO is actually for the Anchor.toml when someone pastes these entries in there (because we don't know how to label them at this point). I thought this was better than writing "Auto-generated" because it doesn't put any onus on the guy to label these after the fact.

Is that all right?

Reply:
ahh ok, that's fine